docker-pushmi-pullyu

```diff
@@ -126,22 +126,22 @@ then
     docker run \
       --detach \
       --volume="docker-pushmi-pullyu:/var/lib/registry" \
-      --publish-all \
+      --publish 127.0.0.1::5000 \
```

Comment:
Nice, I didn't know you could omit the local port number with `--publish` like this. I imagine it binds to a random port like `--publish-all` does?

Reply:
Yes exactly:
```
$ docker port 4bd9b576c5...
5000/tcp -> 127.0.0.1:49159
```